crates/uv-resolver/src/pubgrub/report.rs

```diff
@@ -1530,7 +1530,7 @@ impl std::fmt::Display for PubGrubHint {
                     "{}{} An index URL ({}) could not be queried due to a lack of valid authentication credentials ({}).",
                     "hint".bold().cyan(),
                     ":".bold(),
-                    index.redacted().cyan(),
+                    index.without_credentials().cyan(),
```

Comment:
I'm a bit torn here, on one side seeing `***` here could be confusing cause those credentials are evidently the wrong ones, otoh I'd really want to know that there were credentials on the URL here.

Reply:
Yeah that's a good point. Note that this is not changing the existing behavior.

Reply:
Reflecting more, I think it's helpful this way since it indicates the invalid credentials were on the URL.